tensorflow_probability/python/distributions/sigmoid_beta.py

```diff
@@ -0,0 +1,274 @@
+      allow_nan_stats: Python `bool`, default `True`. When `True`, statistics
+        (e.g., mean, mode, variance) use the value '`NaN`' to indicate the
+        result is undefined. When `False`, an exception is raised if one or
+        more of the statistic's batch members are undefined.
+      name: Python `str` name prefixed to Ops created by this class.
+    """
+    parameters = dict(locals())
+    with tf.name_scope(name) as name:
+      dtype = dtype_util.common_dtype([concentration1, concentration0],
+                                       dtype_hint=tf.float32)
+      self._concentration1 = tensor_util.convert_nonref_to_tensor(
+          concentration1, dtype=dtype, name='concentration1')
+      self._concentration0 = tensor_util.convert_nonref_to_tensor(
+          concentration0, dtype=dtype, name='concentration0')
+
+    super(SigmoidBeta, self).__init__(
+        dtype=dtype,
+        validate_args=validate_args,
+        allow_nan_stats=allow_nan_stats,
+        reparameterization_type=reparameterization.FULLY_REPARAMETERIZED,
+        parameters=parameters,
+        name=name)
+
+  @classmethod
+  def _parameter_properties(cls, dtype, num_classes=None):
+    # pylint: disable=g-long-lambda
+    return dict(
+          concentration1=parameter_properties.ParameterProperties(
+            default_constraining_bijector_fn=(
+              lambda: softplus_bijector.Softplus(low=dtype_util.eps(dtype)))),
+          concentration0=parameter_properties.ParameterProperties(
+            default_constraining_bijector_fn=(
+              lambda: softplus_bijector.Softplus(low=dtype_util.
+                                                 eps(dtype)))))
+    # pylint: enable=g-long-lambda
+
+  @property
+  def concentration1(self):
+    """Concentration parameter associated with a `1` outcome."""
+    return self._concentration1
+
+  @property
+  def concentration0(self):
+    """Concentration parameter associated with a `0` outcome."""
+    return self._concentration0
+
+  def _default_event_space_bijector(self):
+    return identity_bijector.Identity(validate_args=self.validate_args)
+
+  def _batch_shape_tensor(self, concentration1=None, concentration0=None):
+    return ps.broadcast_shape(
+          ps.shape(self.concentration1
+                   if concentration1 is None else concentration1),
+          ps.shape(self.concentration0
+                   if concentration0 is None else concentration0))
+
+  def _batch_shape(self):
+    return tf.broadcast_static_shape(self.concentration1.shape,
+                                     self.concentration0.shape)
+
+  def _parameter_control_dependencies(self, is_init):
+    if not self.validate_args:
+      return []
+    assertions = []
+    for i, concentration in enumerate([self.concentration0,
+                                       self.concentration1]):
+      if is_init != tensor_util.is_ref(concentration):
+        assertions.append(
+          assert_util.assert_positive(
```

Comment:
Same here (add 2 spaces)

Reply:
When I make this change and run `./testing/run_tfp_lints.sh tensorflow_probability/python/distributions/sigmoid_beta.py` locally I get the following error:

```
************* Module tensorflow_probability.python.distributions.sigmoid_beta
W0311:215 : Bad indentation. Found 10 spaces, expected 8 [bad-indentation]
```

Additionally, as written this method currently matches what's in `distributions/beta.py`. Does the `test/pylintrc` file in the OS mirror not match what's used internally?